Pet training/Pet-training.md

```diff
@@ -2,48 +2,58 @@
 ```bash
 {
 "**Prompt:**" "How can I schedule a training session with a pet trainer through Petofy?"
-"**Response:**" "Scheduling a training session through Petofy is so simple. Simply input your location in the designated box, navigate to the pet training section, and choose the best trainer nearby."
+"**Response:**" "To schedule a training session through Petofy please visit https://www.petofy.com/ or our mobile app. Now follow these steps:
+    - Go to Training under Find services.
+    - Select your city from the list.
+    - Look for highly-rated training center for you pet.
+    Make sure are logged into your Petofy account"
 }
 ```
 ```bash
 {
 "**Prompt:**" "What is the training duration for my pet?"
-"**Response:**" "Training durations may vary based on the specific program and your pet's needs. Please contact our help desk for more details."
+"**Response:**" "Training durations may vary based on the specific program and your pet's needs. Please reach out to our customer support at https://www.petofy.com/contact-us or call us at +(91) 6399-638-639 for more detailed assistance"
```

Comment:
depends upon the selected training program and any other training request from pet parent.